qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -453,4 +454,52 @@ def test_snap_schedule_with_mgr_restart(self):
         # cleanup
         self.fs_snap_schedule_cmd('remove', path=testdir, snap_schedule='1M')
         self.remove_snapshots(testdir[1:])
-        self.mount_a.run_shell(['rmdir', testdir[1:]])
+        self.mount_a.run_shell(['rmdir', testdir[1:]])    
+
+class TestSnapSchedulesSnapdir(TestSnapSchedulesHelper):
+    def remove_snapshots(self, dir_path, sdn):
+        snap_path = f'{dir_path}/{sdn}'
+
+        snapshots = self.mount_a.ls(path=snap_path)
+        for snapshot in snapshots:
+            snapshot_path = os.path.join(snap_path, snapshot)
+            log.debug(f'removing snapshot: {snapshot_path}')
+            self.mount_a.run_shell(['rmdir', snapshot_path])
+
+    def get_snap_dir_name(self):
+        from tasks.cephfs.fuse_mount import FuseMount
+        from tasks.cephfs.kernel_mount import KernelMount
+
+        sdn = self.fs.get_config('client_snapdir')
+        if sdn == None:
+            if isinstance(self.mount_a, KernelMount):
+                sdn = self.mount_a.client_config.get('snapdirname', '.snap')
+            elif isinstance(self.mount_a, FuseMount):
+                sdn = self.mount_a.client_config.get('client_snapdir', '.snap')
+                self.fs.set_ceph_conf('client', 'client snapdir', sdn)
+                self.mount_a.umount_wait()
```

Comment:
Here you didn't remount it, you just unmounted the `mount_a`.

`s/self.mount_a.umount_wait()/self.mount_a.remount()/`

Reply:
oops, missed it...Will update